documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -179,28 +179,47 @@ TIP: On multi-core Raspberry Pi models, the `make -j<n>` option distributes work
 [[native-install]]
 ==== Install the kernel
 
-Next, install the kernel, modules, and Device Tree blobs into the boot partition of your Raspberry Pi.
+Next, install the kernel modules onto the boot media:
+
+* For 64-bit kernels:
++
+[source,console]
+----
+$ sudo make -j6 modules_install
+----
+
+* For 32-bit kernels:
++
+[source,console]
+----
+$ sudo make -j6 modules_install
+----
```

Comment:
Should this be collapsed to "For both 64-bit and 32-bit kernels, run:" ? Seems a bit weird to label them separately, if the command in both cases is identical?

Reply:
Good point. I strayed a little too close to dogma here when it came to parallel structure!

Reply:
If only there was some kind of Tab system... :laughing: